optuna/storages/rdb/storage.py

```diff
@@ -671,6 +671,30 @@ def get_all_trials(self, study_id):
         trials = [self.get_trial(trial_id) for trial_id in trial_ids]
         return trials
 
+    def get_best_trial(self, study_id):
+        # type: (int) -> structs.FrozenTrial
+
+        session = self.scoped_session()
+        if self.get_study_direction(study_id) == structs.StudyDirection.MAXIMIZE:
+            trial = models.TrialModel.find_max_value_trial(study_id, session)
+        else:
+            trial = models.TrialModel.find_min_value_trial(study_id, session)
+
+        params = models.TrialParamModel.where_trial(trial, session)
+        values = models.TrialValueModel.where_trial(trial, session)
+        user_attributes = models.TrialUserAttributeModel.where_trial(trial, session)
+        system_attributes = models.TrialSystemAttributeModel.where_trial(trial, session)
+
+        frozen_trial = self._merge_trials_orm([trial], params, values, user_attributes,
+                                              system_attributes)[0]
+
+        self._finished_trials_cache.cache_trial_if_finished(frozen_trial)
```

Comment:
Did you forget to read from the cache, in order to actually make use of it (c.f. `get_trial`)?

Reply:
I replaced the code to`get_trial`

Reply:
done!